sklearnex/tests/test_config.py

```diff
@@ -30,13 +30,17 @@ def test_get_config_contains_sklearn_params():
 def test_set_config_works():
     default_config = sklearnex.get_config()
     sklearnex.set_config(
-        assume_finite=True, target_offload="cpu:0", allow_fallback_to_host=True
+        assume_finite=True,
+        target_offload="cpu:0",
+        allow_fallback_to_host=True,
+        allow_sklearn_after_onedal=False,
```

Comment:
Check generally this test, how is it safe for the config_context work

Reply:
done at https://github.com/intel/scikit-learn-intelex/pull/1907/commits/205289e8758b2cc0f4e1364b348c4f36d6484f99